src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -66,6 +67,8 @@ class ResultsList extends Component {
           // Support non va urgent care search through ccp option
           if (query.serviceType === CLINIC_URGENTCARE_SERVICE) {
             item = <UrgentCareResult provider={r} query={query} key={r.id} />;
+          } else if (query.serviceType === PHARMACY_RETAIL_SERVICE) {
+            item = <PharmacyResult provider={r} query={query} key={r.id} />;
```

Comment:
It would be nice to add a unit test that covers this logic. This could be done in a separate PR.